charts/wadm/values.yaml

```diff
@@ -14,12 +14,14 @@ wadm:
     hostId: ""
     logLevel: ""
     nats:
-      credsFile: ""
-      jetstreamDomain: ""
-      jwt: ""
-      seed: ""
       server: "127.0.0.1:4222"
+      jetstreamDomain: ""
       tlsCaFile: ""
+      creds:
+        jwt:
+        seed:
```

Comment:
Are these keys of no value equivalent to specifying `""`

Reply:
That's correct, and honestly there's no reason to have the double quotes missing here, so I went ahead and added those back for these values as well.